question24/register.html

```diff
@@ -0,0 +1,196 @@
+                    </ol>
+                </div>
+                <div class="modal_content">
+                    <h3>第8条（退会）</h3>
+                    <p>ユーザーは，当社の定める退会手続により，本サービスから退会できるものとします。</p>
+                    <h3>第9条（保証の否認および免責事項）</h3>
+                    <ol>
+                        <li>
+                            当社は，本サービスに事実上または法律上の瑕疵（安全性，信頼性，正確性，完全性，有効性，特定の目的への適合性，セキュリティなどに関する欠陥，エラーやバグ，権利侵害などを含みます。）がないことを明示的にも黙示的にも保証しておりません。
+                        </li>
+                        <li>
+                            当社は，本サービスに起因してユーザーに生じたあらゆる損害について一切の責任を負いません。ただし，本サービスに関する当社とユーザーとの間の契約（本規約を含みます。）が消費者契約法に定める消費者契約となる場合，この免責規定は適用されません。
+                        </li>
+                        <li>
+                            前項ただし書に定める場合であっても，当社は，当社の過失（重過失を除きます。）による債務不履行または不法行為によりユーザーに生じた損害のうち特別な事情から生じた損害（当社またはユーザーが損害発生につき予見し，または予見し得た場合を含みます。）について一切の責任を負いません。また，当社の過失（重過失を除きます。）による債務不履行または不法行為によりユーザーに生じた損害の賠償は，ユーザーから当該損害が発生した月に受領した利用料の額を上限とします。
+                        </li>
+                        <li>当社は，本サービスに関して，ユーザーと他のユーザーまたは第三者との間において生じた取引，連絡または紛争等について一切責任を負いません。</li>
+                    </ol>
+                </div>
+                <div class="modal_content">
+                    <h3>第10条（サービス内容の変更等）</h3>
+                    <p>
+                        当社は，ユーザーに通知することなく，本サービスの内容を変更しまたは本サービスの提供を中止することができるものとし，これによってユーザーに生じた損害について一切の責任を負いません。
+                    </p>
+                </div>
+                <div class="modal_content">
+                    <h3>第11条（利用規約の変更）</h3>
+                    <p>
+                        当社は，必要と判断した場合には，ユーザーに通知することなくいつでも本規約を変更することができるものとします。なお，本規約の変更後，本サービスの利用を開始した場合には，当該ユーザーは変更後の規約に同意したものとみなします。
+                    </p>
+                </div>
+                <div class="modal_content">
+                    <h3>第12条（個人情報の取扱い）</h3>
+                    <p>当社は，本サービスの利用によって取得する個人情報については，当社「プライバシーポリシー」に従い適切に取り扱うものとします。</p>
+                </div>
+                <div class="modal_content">
+                    <h3>第13条（通知または連絡）</h3>
+                    <p>
+                        ユーザーと当社との間の通知または連絡は，当社の定める方法によって行うものとします。当社は,ユーザーから,当社が別途定める方式に従った変更届け出がない限り,現在登録されている連絡先が有効なものとみなして当該連絡先へ通知または連絡を行い,これらは,発信時にユーザーへ到達したものとみなします。
+                    </p>
+                </div>
+                <div class="modal_content">
+                    <h3>第14条（権利義務の譲渡の禁止）</h3>
+                    <p>ユーザーは，当社の書面による事前の承諾なく，利用契約上の地位または本規約に基づく権利もしくは義務を第三者に譲渡し，または担保に供することはできません。</p>
+                </div>
+                <div class="modal_content">
+                    <h3>第15条（準拠法・裁判管轄）</h3>
+                    <ol>
+                        <li>本規約の解釈にあたっては，日本法を準拠法とします。</li>
+                        <li>本サービスに関して紛争が生じた場合には，当社の本店所在地を管轄する裁判所を専属的合意管轄とします。</li>
+                    </ol>
+                </div>
+                <div class="modal_content last_content" id="js-end">
+                    <p>以上</p>
+                </div>
```

Comment:
Q. Is `js-end` used?

Reply:
Thank you for your review! Deleted as not in use. Please confirm.

fixed : https://github.com/csakyo/homework/pull/38/commits/77f8b948e8a5b5798c5610e07a3c88191f845d93